app/src/components/Day.js

```diff
@@ -7,80 +7,59 @@ import ListGroupItem from "react-bootstrap/ListGroupItem";
 
 import { moonPhases } from "../MoonPhases";
 import * as apiClient from "../apiClient";
-
-const Day = (data) => {
+//rename data to props
```

Comment:
Is this a TODO? If so, the general standard in JS and other languages is to mark it as `TODO` ie:
```suggestion
// TODO: rename data to props
```